tests/plugins/test_check_base_image.py

```diff
@@ -123,53 +114,51 @@ def test_pull_base_image_special(add_another_parent, special_image, change_base,
 
 @pytest.mark.skip(reason="plugin needs rework to not pull and local tag images")
```

Comment:
Some unit tests are still marked to skip, IMO that needs to be resolved

Reply:
updated accordingly.